library/worldgen/dimension/src/testmod/java/org/quiltmc/qsl/worldgen/dimension/QuiltDimensionTest.java

```diff
@@ -108,10 +107,6 @@ private int swapTargeted(CommandContext<ServerCommandSource> context) throws Com
 			var target = new TeleportTarget(new Vec3d(0.5, 101, 0.5), Vec3d.ZERO, 0, 0);
 			QuiltDimensions.teleport(player, modWorld, target);
 
-			if (player.getWorld() != modWorld) {
```

Comment:
Why was this removed?

Reply:
I removed it because at the time I was unaware of the alternatives to the previously used `CommandException`. It will be re-added with an updated exception type.